[terra-scroll] Added realistic examples

AH106586Harika opened the pull request:
<!-- 
PLEASE COMPLETE THESE STEPS BEFORE PUBLISHING THE PULL REQUEST:

*Before publishing*
1. Update the title as: [component] changes to component (ex. [terra-button] Added terra-button accessibility guide).
2. Fill out the fields below.
3. Assign yourself to the PR.
4. Add the appropriate package name labels.
5. Add your name to the CONTRIBUTORS.md file. Adding your name to the CONTRIBUTORS.md file signifies agreement to all rights and reservations provided by the License.
-->

### Summary
<!--- Added realistic examples  -->

**What was changed:**
updated 'terra-scroll' to add realistic content with interactive elements.


**Why it was changed:**
As the examples are generic changed to realistic examples.

### After Implementation

<img width="1264" alt="image" src="https://github.com/cerner/terra-core/assets/126655256/f68d4ca4-0b79-4ff8-82ef-fa009c7620b2">



### Testing
<!-- Demonstrate that these changes are stable. How have these changes been verified? -->

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [x] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:
<!-- Please include the appropriate "Required" & "Ready" labels (ex. "UX Review Required" and "UX Review Ready").  -->
- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review


**This PR resolves:**

[UXPLATFORM-9462](https://jira2.cerner.com/browse/UXPLATFORM-9462)

---

Thank you for contributing to Terra.
@cerner/terra

supreethmr commented on packages/terra-core-docs/CHANGELOG.md:
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+* Updated
+  * Updated `terra-scroll` to add realistic examples.
```
```suggestion

* Updated
  * Updated `terra-scroll` to add realistic examples.
```

AH106586Harika replied:
Done.

adoroshk reviewed (commented):
Before the change the vertical scroll example had only vertical scroll and horizontal scroll example had only horizontal scroll. When I tested this change, both examples had both, horizontal and vertical scrolls. It might be an expected behavior for the horizontal scroll in case of content being bigger than container, but doesn't look like expected behavior for the vertical scroll container to have a horysontal scroll. Please correct me if I am wrong. 

Also it might look better if the input and button have same padding ad paragraph for consistancy.

<img width="455" alt="Screenshot 2023-08-14 at 9 45 05 AM" src="https://github.com/cerner/terra-core/assets/119358186/60e02d51-8ecc-4b25-890e-bb8f75667fa4">
<img width="437" alt="Screenshot 2023-08-14 at 9 48 18 AM" src="https://github.com/cerner/terra-core/assets/119358186/ece97b4f-1697-40a6-98a1-c74afdef2d37">

AH106586Harika commented:
> Before the change the vertical scroll example had only vertical scroll and horizontal scroll example had only horizontal scroll. When I tested this change, both examples had both, horizontal and vertical scrolls. It might be an expected behavior for the horizontal scroll in case of content being bigger than container, but doesn't look like expected behavior for the vertical scroll container to have a horysontal scroll. Please correct me if I am wrong.
> 
> Also it might look better if the input and button have same padding ad paragraph for consistancy.
> 
> <img alt="Screenshot 2023-08-14 at 9 45 05 AM" width="455" src="https://user-images.githubusercontent.com/119358186/260465443-60e02d51-8ecc-4b25-890e-bb8f75667fa4.png"> <img alt="Screenshot 2023-08-14 at 9 48 18 AM" width="437" src="https://user-images.githubusercontent.com/119358186/260465764-ece97b4f-1697-40a6-98a1-c74afdef2d37.png"


> Before the change the vertical scroll example had only vertical scroll and horizontal scroll example had only horizontal scroll. When I tested this change, both examples had both, horizontal and vertical scrolls. It might be an expected behavior for the horizontal scroll in case of content being bigger than container, but doesn't look like expected behavior for the vertical scroll container to have a horysontal scroll. Please correct me if I am wrong.
> 
> Also it might look better if the input and button have same padding ad paragraph for consistancy.
> 
> <img alt="Screenshot 2023-08-14 at 9 45 05 AM" width="455" src="https://user-images.githubusercontent.com/119358186/260465443-60e02d51-8ecc-4b25-890e-bb8f75667fa4.png"> <img alt="Screenshot 2023-08-14 at 9 48 18 AM" width="437" src="https://user-images.githubusercontent.com/119358186/260465764-ece97b4f-1697-40a6-98a1-c74afdef2d37.png">



Removed horizontal scroll for vertical scroll and reduced content, also removed vertical scroll for horizontal scroll, added padding as suggested for input and submit

adoroshk reviewed (changes requested):
I suggest simplifying the `Scroll` content to be:

<div className={cx('paragraph-x-wrapper')}>
    <p>
      Entering patient details is a crucial process in healthcare facilities that involves capturing and documenting relevant information about
      individuals seeking medical care. Patient details need to be regularly updated to ensure accuracy and relevancy. This may involve periodic
      verification of personal information, updating medical history, and making necessary changes to insurance or contact details.
    </p>
    <InputField
        inputId="input-field"
        label="Enter Patient Name"
        type="text"
    />
    <Button text="Submit" />
</div>

In this case we don't need `container-x-wrapper` class.

kademswetha commented:
+1
